_posts/2022-02-23-sparse-models.md

```diff
@@ -5,14 +5,13 @@ tags: []
 categories: [Machine Learning]
 layout: post
 authors: [ojus1]
+latex: True
 ---
 
-<script src='https://cdn.plot.ly/plotly-2.8.3.min.js'></script>
-
-The most fancy tool in the Modern Machine Learning toolbox are Neural Networks (NNs),
-especially `Deep` ones. NNs usually have much more number of parameters than the
-number of datapoints, and hence are `overparameterized` models. Are all of the parameters
-needed and perform a useful function in the model?
+The most fancy tool in the Modern Machine Learning toolbox are Neural Networks (NNs), 
+especially `Deep` ones. NNs usually have much more number of parameters than the 
```

Comment:
Minor niggle, can we use fixed fonts only for code? You can use emphasis for _Deep_ here.

Reply:
Fixed